src/test/commands/createPropertiesFile.test.ts

```diff
@@ -70,7 +70,7 @@ describe('Create Properties File', () => {
         await createPropertiesFile();
         expect(inputBoxSpy.calledOnce).to.be.true;
         expect(inputBoxSpy()).to.be.undefined;
-    });
+    }).timeout(10000);
 
```

Comment:
Why is there a 10s timeout on this? It will cause a huge delay in running tests.

Reply:
Done. I have reduced the timeout to 1s

Reply:
So I think the issue is here Paul that Mocha needs to be explicitly told how long to wait for before it will declare a test broken. Finding the right balance between testing and speed of running the test suite is hard. Too long and we slow everything down, too fast and we might introduce a fragility in the test that doesn't echo real-world code execution. 

https://www.testim.io/blog/testing-promises-using-mocha/